src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int HUNDRED = 100;
+    private ColorSupplier supplier = new ColorSupplier();
+    private int random = new Random().nextInt(HUNDRED);
+
+    public String getRandomBall() {

```

Comment:
This method should return Ball but return String. For printing ball characteristics you should override method `toString() ` in class Ball